### ISSUE TITLE
Color for bigbed track

### ISSUE THREAD

**Issue by dnil** [igvteam/igv.js#1646]
The starting coloration for bigbed tracks appears to have turned "blue" sometime around the switch from IGV.js 2.15.3 to 2.15.4. BB file annotation track coloration can be restored by selecting "Unset track color". I guess this was unintentional from the new nice custom track coloration options. Is there perhaps an option we could use to auto-unset? Thanks a bunch!

**Comment by jrobinso** [comment]
That would be a bug,  However how is the color specified in your BB file?   Individually per feature?

**Comment by dnil** [comment]
Yes, exactly, as per BED coloring. This UCSC ClinVar track is representative: https://hgdownload.soe.ucsc.edu/gbdb/hg19/bbi/clinvar/clinvarMain.bb

A corresponding config snippet:

```
                        {
                            name: "ClinVar",
                            type: "annotation",
                            format: "bigBed",
                            sourceType: "file",
                            url: "https://hgdownload.soe.ucsc.edu/gbdb/hg19/bbi/clinvar/clinvarMain.bb",
                            displayMode: "EXPANDED",
                            visibilityWindow: 300000,
                            order: Number.MIN_VALUE,                            
                        },
```

**Comment by dnil** [comment]
❌ A screenshot from directly after loading with 2.15.6:
![Screenshot 2023-05-26 at 09 48 56](https://github.com/igvteam/igv.js/assets/758570/8963eea6-35d8-44e5-ae43-cb3852472b8c)

✅ After selecting "Unset track color" (2.15.6):
![Screenshot 2023-05-26 at 09 49 05](https://github.com/igvteam/igv.js/assets/758570/b889263e-3bbd-40b8-a900-126887a26cc5)

✅ Directly after load with 2.15.3:
![Screenshot 2023-05-26 at 09 49 40](https://github.com/igvteam/igv.js/assets/758570/83f431a8-6b3a-497f-bba9-3175c767abb0)

❌ Directly after load with 2.15.4:
![Screenshot 2023-05-26 at 09 52 39](https://github.com/igvteam/igv.js/assets/758570/823b7bc9-f16b-4384-aeec-56cebe19a4fa)

**Comment by jrobinso** [comment]
This should be fixed now with release 2.15.8
